layers/+lang/emacs-lisp/packages.el

```diff
@@ -159,7 +159,12 @@
 
 (defun emacs-lisp/init-elisp-def ()
   (use-package elisp-def
-    :defer t))
+    :defer t
+    :commands (elisp-def)
```

Comment:
This command is already autoloaded by the package itself so this line is redundant.  LGTM otherwise

Reply:
Sure, removed this line on the latest change. Thanks